backend/parameter_store.py

```diff
@@ -1,20 +1,46 @@
+# FIXME: rename file to parameters.py
+
 import json
 import logging
+from enum import Enum
 from typing import TYPE_CHECKING
 
-if TYPE_CHECKING:
-    # When type checking we want to use the third party package's stub
-    from mypy_boto3_ssm import SSMClient
-else:
-    # In production we want to avoid depending on a package which has no runtime impact
-    SSMClient = object
+import boto3
+
+from .environment import ENV
+
+ssm_client = boto3.client("ssm")
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
 
+# if TYPE_CHECKING:
+#     # When type checking we want to use the third party package's stub
+#     from mypy_boto3_ssm import SSMClient
+# else:
+#     # In production we want to avoid depending on a package which has no runtime impact
+#     SSMClient = object
 
-def get_param(parameter: str, ssm_client: SSMClient, logger: logging.Logger) -> str:
+
+# def get_param(parameter: str, ssm_client: SSMClient, logger: logging.Logger) -> str:
+def get_param(parameter: str) -> str:
```

Comment:
get_param could take as input (parameter: ParameterName) and then we could pass in ParameterName.example instead of ParameterName.example.value everywhere we use it.